src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator/Calculator';
 
 function App() {
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/164142909-8d7df155-529f-4d11-9d6b-bd945960b700.png)

Therefore:

- [ ] In order to remove all the CRA boilerplate that is no longer used, kindly delete the following files:

  - App.test.js
  - index.css
  - logo.svg
  - reportWebVitals.js
  - setupTests.js
  - public/logo192.png
  - public/logo512.png
  - public/manifest.json
  - public/robots.txt

That way, we will have a cleaner repo without unused files! 💪